### ISSUE TITLE
Add ignoreAtRules: [] to no-invalid-position-at-import-rule

### ISSUE THREAD

**Issue by GrimLink** [stylelint/stylelint#5492]
It would be nice to have an option to keep the `no-invalid-position-at-import-rule` on and also support `@use`.
This syntax is used by both Sass but some PostCSS plugins also use at rules in the CSS directly.
Adding this option give some flexibility.

Not a must for me, but it would be a nice to have.

**Comment by ybiquitous** [comment]
@GrimLink Can you check #5322, please?

**Comment by GrimLink** [comment]
@ybiquitous I did 😉 
And the related tickets.

I am not having an issue with it at the moment,
since I just disabled the rule `no-invalid-position-at-import-rule` in my scss config.

I am just asking for an option so I can keep `no-invalid-position-at-import-rule` enabled,
and also allow other at-rules on top.

**Comment by ybiquitous** [comment]
I see. Is it possible to use `/* stylelint-disable */` inline comments in your case?

For example:

```scss
/* stylelint-disable-next-line no-invalid-position-at-import-rule */
@use "...";

@import "...";
```

See also <https://stylelint.io/user-guide/ignore-code/#within-files>

**Comment by GrimLink** [comment]
@ybiquitous it is for me,
but I also work in a team.
And my colleges only use stylelint and don't configure it or exclude rules.
Which you want to avoid to keep consistency.

I was just wondering this could be an option, as I already mentioned in my previous comments

**Comment by Semigradsky** [comment]
Ideally it would be to create `sass/no-invalid-position-at-import-rule` but option would be nice workaround.

**Comment by ybiquitous** [comment]
@Semigradsky Thanks for the help. As you commented, it seems ideal also to me to add a new rule to the `stylelint-scss` plugin like [`scss/at-rule-unknown`](https://github.com/kristerkari/stylelint-scss/tree/7c0f68cfb154ed790a6c79e6cc959281b678ffca/src/rules/at-rule-no-unknown). The rule handles [`@use`](https://github.com/kristerkari/stylelint-scss/blob/7c0f68cfb154ed790a6c79e6cc959281b678ffca/src/rules/at-rule-no-unknown/index.js#L23) internally.

**Comment by jeddy3** [comment]
@GrimLink Thanks for the request.

> Adding this option give some flexibility.

I agree. The primary purpose of secondary options is to make rules more permissive. We can add an `ignoreAtRules: []` secondary option to the rule.

As @Semigradsky and @ybiquitous suggested, this would eventually make it into `stylelint-scss` as `scss/at-import-no-invalid-position`, which would make use of this new `ignoreAtRules: []` secondary option and the [`checkAgainstRule` util](https://stylelint.io/developer-guide/plugins#stylelintutilscheckagainstrule). 

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to add a new option](https://stylelint.io/developer-guide/rules#add-an-option-to-a-rule) in the Developer guide.

Please branch of `v14` rather than `master`.

**Comment by jeddy3** [comment]
Done in #5520
